common/views/components/VenueClosedPeriods/VenueClosedPeriods.js

```diff
@@ -0,0 +1,64 @@
+import { useContext } from 'react';
+
+import {
+  backfillExceptionalVenueDays,
+  getExceptionalOpeningPeriods,
+  getExceptionalClosedDays,
+  groupConsecutiveDays,
+  convertJsonDateStringsToMoment,
+} from '../../../services/prismic/opening-times';
+import { formatDayDate } from '@weco/common/utils/format-date';
+import OpeningTimesContext from '@weco/common/views/components/OpeningTimesContext/OpeningTimesContext';
+
+type Props = {|
+  venue: any, // FIXME: Flow
+|};
+
+const VenueClosedPeriods = ({ venue }: Props) => {
+  const openingTimes = useContext(OpeningTimesContext);
+  const exceptionalPeriods = getExceptionalOpeningPeriods(openingTimes);
+  const backfilledExceptionalPeriods = backfillExceptionalVenueDays(
+    convertJsonDateStringsToMoment(venue),
+    exceptionalPeriods
+  );
+  const onlyClosedDays =
+    backfilledExceptionalPeriods &&
+    getExceptionalClosedDays(backfilledExceptionalPeriods);
+  const groupedConsectiveClosedDays = onlyClosedDays
+    ? groupConsecutiveDays(onlyClosedDays)
+    : [];
+
+  return groupedConsectiveClosedDays[0].length > 0 ? (
```

Comment:
When accessing the first in an array, we have had this break pages, as well as core bits of the site, is it worth being conservative here?

Reply:
👍